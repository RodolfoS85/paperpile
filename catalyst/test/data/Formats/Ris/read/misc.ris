TY  - JOUR
AU  - Waterman, M. S.
AU  - Smith, T. F.
PY  - 1978//
TI  - RNA secondary structure: A complete mathematical analysis
JO  - Mathematical Biosciences
SP  - 257
EP  - 266
VL  - 42
ER  - 

TY  - JOUR
T1  - Radar penetrates only the youngest geological units on Mars
A1  - Stillman, David E.
A1  - Grimm, Robert E.
Y1  - 2011/03/03
JO  - J. Geophys. Res.
VL  - 116
IS  - E3
SP  - E03001
SN  - 0148-0227
KW  - Mars
KW  - SHARAD
KW  - adsorbed water
KW  - scattering
KW  - GPR
KW  - 6225 Planetary Sciences: Solar System Objects: Mars
KW  - 5464 Planetary Sciences: Solid Surface Planets: Remote sensing
KW  - 5460 Planetary Sciences: Solid Surface Planets: Physical properties of materials
KW  - 3914 Mineral Physics: Electrical properties
KW  - 0669 Electromagnetics: Scattering and diffraction
N2  - Signals from the Shallow Radar were intended to penetrate hundreds of meters or more into Mars...
UR  - http://dx.doi.org/10.1029/2010JE003661
DO  - 10.1029/2010JE003661
PB  - AGU
ER  - 

TY  - JOUR
A1  - Spotorno, S
A1  - Faure, S
Y1  - 2011
T1  - Change detection in complex scenes: Hemispheric contribution and the role of perceptual and semantic factors
JO  - Perception
SP  - 5
EP  - 22
VL  - 40
IS  - 1
UR  - http://www.perceptionweb.com/abstract.cgi?id=p6524
PB  - Pion Ltd
SN  - 
N2  - The perceptual salience and semantic relevance of objects for the meaning of a scene were evaluated with multiple criteria...
ER  - 

TY  - ABST
ID  - S0092-8674(11)00127-9
DOI  - 10.1016/j.cell.2011.02.013
AU  - Hanahan, Douglas
AU  - Weinberg, Robert A.
T1  - Hallmarks of Cancer: The Next Generation
Y1  - 2011/03/04
JF  - Cell
JO  - Cell
JA  - Cell
VL  - 144
IS  - 5
SP  - 646
EP  - 674
PB  - Cell Press
SN  - 0092-8674
UR  - http://linkinghub.elsevier.com/retrieve/pii/S0092867411001279
ER  - 

