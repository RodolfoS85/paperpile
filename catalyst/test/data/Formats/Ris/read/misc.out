---
authors: 'Waterman, M. S. and Smith, T. F.'
journal: Mathematical Biosciences
pages: 257-266
pubtype: ARTICLE
title: 'RNA secondary structure: A complete mathematical analysis'
volume: 42
year: 1978
---
authors: 'Stillman, David E. and Grimm, Robert E.'
title: 'Radar penetrates only the youngest geological units on Mars'
journal: 'J. Geophys. Res.'
day: 03
month: 03
year: 2011
volume: 116
issue: E3
pages: E03001
issn: 0148-0227
abstract: 'Signals from the Shallow Radar were intended to penetrate hundreds of meters or more into Mars...'
url: http://dx.doi.org/10.1029/2010JE003661
doi: 10.1029/2010JE003661
publisher: AGU
---
authors: 'Spotorno, S and Faure, S'
title: 'Change detection in complex scenes: Hemispheric contribution and the role of perceptual and semantic factors'
journal: 'Perception'
year: 2011
volume: 40
pages: 5-22
issue: 1
abstract: 'Signals from the Shallow Radar were intended to penetrate hundreds of meters or more into Mars...'
url: http://www.perceptionweb.com/abstract.cgi?id=p6524
publisher: Pion Ltd
---
authors: 'Hanahan, Douglas and Weinberg, Robert A.'
title: 'Hallmarks of Cancer: The Next Generation'
year: 2011
month: 03
day: 04
journal: 'Cell'
volume: 144
issue: 5
pages: 646-674
publisher: 'Cell Press'
pubtype: ARTICLE
issn: 0092-8674
url: http://linkinghub.elsevier.com/retrieve/pii/S0092867411001279
doi: 10.1016/j.cell.2011.02.013
---
authors: 'Toulmin, Camilla'
title: 'Agriculture: A bowl half full'
pubtype: ARTICLE
journal: 'Nature'
year: 2011
month: 03
day: 03
volume: 471
issue: 7336
pages: 31-32
publisher: 'Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved.'
issn: 0028-0836
url: http://dx.doi.org/10.1038/471031a
doi: 10.1038/471031a

